makeflow/src/makeflow.c

```diff
@@ -1109,7 +1138,7 @@ static void makeflow_run( struct dag *d )
 
 	/* Always make final report to catalog when workflow ends. */
```

Comment:
Make sure to emit a final status file when the workflow is done, so that the status file reflects all jobs complete.